src/consensus/pbft/libbyz/parameters.h

```diff
@@ -12,7 +12,7 @@ const int Max_num_replicas = 32;
 // Interval in sequence space between "checkpoint" states, i.e.,
 // states that are checkpointed and for which Checkpoint messages are
 // sent.
-const int checkpoint_interval = 256;
```

Comment:
How does this relate to sig_max_tx/ms? Is the batch size for the pre-prepares?

Reply:
This is how often we flush our internal data structures. This all changes will Olga's work of when we recover the ledger.